site/_resources/upgrading.md

```diff
@@ -54,6 +54,14 @@ You'll need to re-check where your DNS names are pointing as well, using [Get yo
 **Note:** If you deployed Contour into a different namespace than `projectcontour` with a standard example, please delete that namespace.
 Then in your editor of choice do a search and replace for `projectcontour` and replace it with your preferred name space and apply the updated manifest.
 
+**Note:** If you are deploying to a cluster where you have previously installed alpha versions of the Contour API, applying the Contour CRDs may fail with a message similar to `Invalid value: "v1alpha1": must appear in spec.versions`. In this case, you need to delete the old CRDs before applying the new ones.
+
+```
+$ kubectl delete namespace projectcontour
+$ kubectl get crd  | awk '/projectcontour.io|contour.heptio.com/{print $1}' | xargs kubectl delete crd
```

Comment:
Not a big deal but I don't think this problem affects the `contour.heptio.com` crds?

It might be good to add a note that doing this will delete any CRDs existing in the cluster, so essentially, you'll have to reapply all your configuration again.

Reply:
I don’t really remember which of the four CRDs are affected, only that both `examples/contour` and `examples/render` have them (as said above), I’ll update this to be more explicit about it.